ambry-tools/src/main/java/com.github.ambry/store/ReplicaToBlobIdConverter.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2018 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.clustermap.ClusterMap;
+import com.github.ambry.config.StoreConfig;
+import com.github.ambry.utils.Pair;
+import com.github.ambry.utils.Throttler;
+import com.github.ambry.utils.Time;
+import java.io.File;
+import java.util.Map;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Takes an array of replica directories, scans them for store keys,
+ * then runs the store keys through a store key converter
+ */
+public class ReplicaToBlobIdConverter {
```

Comment:
naming has to be better (see comment below about class not being necessary - can brainstorm names if you don't agree with that comment)